client/src/components/Approval.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Confirmation from './Confirmation';
+
+function Approval(approved) {
```

Comment:
Why is the name of function in UpperCamelCase?

Reply:
This doesn't look like a React Component